src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorFunctionalTest.php

```diff
@@ -236,6 +236,34 @@ public function testConstraintsInDifferentGroupsOnSingleField()
         $this->assertInstanceOf(Length::class, $errors[0]->getCause()->getConstraint());
     }
 
+    public function testConstraintsInDifferentGroupsOnSingleFieldWithAdditionalFieldThatHasNoConstraintsAddedBeforeTheFieldWithConstaints()
```

Comment:
there's a typo in the last `constraints` (BeforeTheFieldWith**Constaints**)

Reply:
Type should be resolved now